### PR TITLE
Fleshing out JOSS COI policy for authors

### PR DESCRIPTION
**Pull request by arfon**
EOM

### CODE REVIEW

File: docs/submitting.md [hunk 1]
```diff
@@ -54,10 +54,28 @@ Authors wishing to publish software deemed out of scope for JOSS have a few opti
...
 While we are happy to review submissions in standalone repositories, we also review submissions that are significant contributions made to existing packages. It is often better to have an integrated library or package of methods than a large number of single-method packages.
 
-### Questions? Open an issue to ask
+### Questions? Open an issue to ask.
 
 Authors wishing to make a pre-submission enquiry should [open an issue](https://github.com/openjournals/joss/issues/new?title=Pre-submission%20enquiry) on the JOSS repository.
 
+## Conflict of Interest policy for authors
+
+Conflict of interest (COI) arises when an author has financial, personal, or other interests that may influence their research or the interpretation of its results. In order to maintain the integrity of the work published in JOSS, we require that authors disclose any potential conflicts of interest at submission time.
```
Comment (danielskatz) [inline docs/submitting.md:63]:
```suggestion
An author conflict of interest (COI) arises when an author has financial, personal, or other interests that may influence their research or the interpretation of its results. In order to maintain the integrity of the work published in JOSS, we require that authors disclose any potential conflicts of interest at submission time.
```

File: docs/submitting.md [hunk 2]
```diff
@@ -54,10 +54,28 @@ Authors wishing to publish software deemed out of scope for JOSS have a few opti
...
+
+### Policy
+
+**Disclosure:** All authors must disclose any potential conflicts of interest related to the research in their manuscript, including financial, personal, or professional relationships that may affect their objectivity. This includes any financial relationships, such as employment, consultancies, honoraria, stock ownership, or other financial interests that may be relevant to the research.
+
+**Acknowledgement:** Authors should acknowledge all sources of financial support for the research and include a statement indicating whether or not the sponsor had any involvement in the study design, data collection, analysis, or interpretation of the results.
```
Comment (danielskatz) [inline docs/submitting.md:69]: JOSS papers aren't supposed to have results, so I wonder if this is needed, or can be better phrased for JOSS

File: docs/submitting.md [hunk 3]
```diff
@@ -54,10 +54,28 @@ Authors wishing to publish software deemed out of scope for JOSS have a few opti
...
+
+**Review process:** Editors and reviewers must be informed of any potential conflicts of interest before reviewing the manuscript to ensure unbiased evaluation of the research.
+
+**Mitigation of COI:** If a potential conflict of interest is identified, steps will be taken to mitigate the influence of the conflict, including but not limited to, disclosure in the manuscript and adjustment of the author's role in the research or removal of the author from the research entirely.
```
Comment (danielskatz) [inline docs/submitting.md:73]: This doesn't make sense - how could we adjust the author's role i the research or remove the author from the research?

File: docs/submitting.md [hunk 4]
```diff
@@ -54,10 +54,28 @@ Authors wishing to publish software deemed out of scope for JOSS have a few opti
...
+
+**Compliance:** Authors who fail to comply with the COI policy may have their manuscript rejected or retracted if it is discovered after publication.
```
Comment (danielskatz) [inline docs/submitting.md:75]:
```suggestion
**Compliance:** Authors who fail to comply with the COI policy may have their manuscript rejected or retracted if a conflict is discovered after publication.
```

File: docs/submitting.md (hunk 3 above)
Comment (arfon) [reply docs/submitting.md:73]: Yeah, sorry. I meant to edit this bit more before pushing the changes up. I'll remove this entirely.

File: docs/submitting.md (hunk 2 above)
Comment (arfon) [reply docs/submitting.md:69]:
```suggestion
**Acknowledgement:** Authors should acknowledge all sources of financial support for the research and include a statement indicating whether or not the sponsor had any involvement in the study design, data collection, analysis, or interpretation of the work.
```

File: docs/submitting.md [hunk 5]
```diff
@@ -54,10 +54,26 @@ Authors wishing to publish software deemed out of scope for JOSS have a few opti
...
 
+## Conflict of Interest policy for authors
+
+An author conflict of interest (COI) arises when an author has financial, personal, or other interests that may influence their research or the interpretation of its results. In order to maintain the integrity of the work published in JOSS, we require that authors disclose any potential conflicts of interest at submission time.
+
+### Policy
+
+**Disclosure:** All authors must disclose any potential conflicts of interest related to the research in their manuscript, including financial, personal, or professional relationships that may affect their objectivity. This includes any financial relationships, such as employment, consultancies, honoraria, stock ownership, or other financial interests that may be relevant to the research.
+
+**Acknowledgement:** Authors should acknowledge all sources of financial support for the research and include a statement indicating whether or not the sponsor had any involvement in the study design, data collection, analysis, or interpretation of the work.
```
Comment (arfon) [inline docs/submitting.md:69]: @danielskatz – I've remove 'results' here and changed it for 'work'. It's very likely that the software will have been developed as part of some broader research program so it seems OK to allude to that here?

Comment (danielskatz) [reply docs/submitting.md:69]: I don't think these specifics (study design, data collection, analysis, or interpretation) make sense for JOSS still.  How about just "Authors should acknowledge all sources of financial support for the work and include a statement indicating whether or not the sponsor had any involvement in it."

Comment (arfon) [inline docs/submitting.md:69]:
```suggestion
**Acknowledgement:** Authors should acknowledge all sources of financial support for the work and include a statement indicating whether or not the sponsor had any involvement in it.
```

**Comment by arfon** [comment]
Thanks for the quick feedback @danielskatz!

**Pull request merged**